test/test_oversized_bag.cpp

```diff
@@ -0,0 +1,123 @@
+// Copyright 2019-2021 Lawrence Livermore National Security, LLC and other YGM
+// Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: MIT
+
+#undef NDEBUG
+
+#include <set>
+#include <string>
+#include <vector>
+#include <ygm/comm.hpp>
+#include <ygm/container/oversized_bag.hpp>
+#include <ygm/random.hpp>
+
+int main(int argc, char** argv) {
+  ygm::comm world(&argc, &argv);
+
+  // Test basic tagging
+  {
+    ygm::container::oversized_bag<std::string> bbag(world);
```

Comment:
This is more of a nitpick but I personally believe bag(world) should have its name changed to something like obag(world) or overbag(world) to clarify it is using the oversized bag and not a regular bag.